schema/pharmgkbSchema.mcf

```diff
@@ -0,0 +1,145 @@
+Node: dcid:ChemicalCompoundGeneAssociation
+name: "ChemicalCompoundGeneAssociation"
+typeOf: schema:Class
+subClassOf: schema:Intangible
+description: "The association of a chemical compound with a gene."
+
+Node: dcid:compoundID
+name: "compoundID"
+typeOf: schema:Property
+rangeIncludes: dcs:Drug
+domainIncludes: dcs:ChemicalCompoundGeneAssociation
+
+Node: dcid:geneID
+name: "geneID"
+typeOf: schema:Property
+rangeIncludes: dcs:Gene
+domainIncludes: dcs:ChemicalCompoundGeneAssociation
+
+Node: dcid:pubMedID
+name: "pubMedID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:ChemicalCompoundGeneAssociation
```

Comment:
Looks like these are already defined. We should remove these and other nodes that are already defined in schema/ MCF files?